libs/vm/include/vm/module.hpp

```diff
@@ -475,6 +475,16 @@ class Module
     return deserialization_constructors_;
   }
 
+  void EnableTestAnnotations()
+  {
+    test_annotations_ = true;
+  }
```

Comment:
Does it make sense for the Module to know about test annotations? All it does is pass this Boolean to the Analyser. A simpler way would be to just inject the Boolean as an argument to the Compiler's constructor, where you could pass it to `Analyser::Initialise` directly.